Tasks/Anikeeva/Task5/Task5.py

```diff
@@ -0,0 +1,42 @@
+string = "The quick Brown Fox"
+if len(string) < 1:
+        print("Enter something")
+else:
+    def check_str(string):
```

Comment:
I explicitly stated on the lesson that function must not be declared inside any other operators without any specific reason and you do not have such reason, it's a huge mistake actually, please do not do this next time